content/events/1v1-bullet.md

```diff
@@ -0,0 +1,22 @@
+@title 1v1 Bullet
+
+# 1v1 Bullet
+
+Players will be divided into brackets to play 1v1 fast matches, streamed live, with two minute time limits. All matches played in one sitting. Single elimination so winners play winners. 
+
+## The rules
+
+- Single elimination (no loser bracket).
+- All matches will take place on tiny drylands maps.
+- All participants must be in a VC channel - hop into one of the bullet tourney VC channels as soon as your game starts.  Players must stream their screen into the VC. (It is recommended to turn on Do Not Disturb to stop potentially private messages popping up in the stream)
+- 2 minutes allowed per turn.
+- Hard cap of 40 minutes per match. The winner of a match will be decided based on capitals owned, and then score.
+- Tribes will be assigned to each round. 
+- Create the game in Polytopia with a 5 min turn timer. If either side times out, instant kicking is allowed (as they will have have had over double the time allotted for the turn).
+
+## Notes
+
+- Teams may send multiple competitors to this event
+- Mobile only
```

Comment:
This can be played on steam if both players have steam, but mobile is required. 
```suggestion
- Mobile polytopia required, but may be played on steam if both players in a match have it.
```